src/game.py

```diff
@@ -27,24 +27,21 @@ def get_prompt(self, total_marks_on_board):
         else:
             message.display_prompt_message_for_move(self, current_player)
 
-    def get_prompt_for_occupied_spot(self, board, user_input):
-        if self.new_board.is_spot_taken(board, user_input):
-            message.display_spot_taken_message(self)
+    def get_prompt_for_occupied_spot(self):
+        message.display_spot_taken_message(self)
```

Comment:
what is the benefit to wrapping this method? since we are importing Message as message, why not just call it directly? (this is not a leading question)

Reply:
This is a great call-out!  I think I did it because I wrapped my other two message prompts in a function, `get_prompt`, but that's because they are conditionally running.  This one isn't, so I can extract the call and just put it directly where it's being called.  Will fix now.

Reply:
IF you pass in a Message as a parameter, there could be a benefit though; think about what it could be.

Reply:
don’t let it stop you from merging this though.